cmd/kubeless/function/function.go

```diff
@@ -44,11 +47,28 @@ var FunctionCmd = &cobra.Command{
 	Short: "function specific operations",
 	Long:  `function command allows user to list, deploy, edit, delete functions running on Kubeless`,
 	Run: func(cmd *cobra.Command, args []string) {
+		cli := utils.GetClientOutOfCluster()
+		var lr = langruntime.New(cli, "kubeless", "kubeless-config")
+		lr.ReadConfigMap()
+
+		getServerConfig, err := cmd.Flags().GetBool("get-server-config")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		if getServerConfig == true {
+			logrus.Info("Current Server Config:")
+			logrus.Infof("Supported Runtimes are: %s",
+				strings.Join(lr.GetRuntimes(), ", "))
+			return
+		}
+
 		cmd.Help()
 	},
 }
 
 func init() {
```

Comment:
We should document how to get the runtime info. Right now probably the best place will be `./docs/runtimes.md` or even in the main README (while it explains how to deploy a function)

Reply:
Done.